Gemfile

```diff
@@ -15,7 +15,7 @@ end
 
 gem 'fog', :require => false
 gem 'guid', :require => false
-gem 'google-api-client', :require => false
+gem 'google-api-client', "~> 0.8.7", :require => false
```

Comment:
Well, I fixed the gem version failure, but not the spec test failure.

On ruby 1.9.3:
```
/home/melissa/.rbenv/versions/1.9.3-p551/lib/ruby/gems/1.9.1/gems/fog-profitbricks-2.0.1/lib/fog/profitbricks.rb:4:in `<top (required)>': undefined local variable or method `__dir__' for main:Object (NameError)
	from /home/melissa/.rbenv/versions/1.9.3-p551/lib/ruby/gems/1.9.1/gems/fog-1.38.0/lib/fog.rb:51:in `require'
	from /home/melissa/.rbenv/versions/1.9.3-p551/lib/ruby/gems/1.9.1/gems/fog-1.38.0/lib/fog.rb:51:in `<top (required)>'
	from /home/melissa/puppetlabs-cloud_provisioner/lib/puppet/cloudpack.rb:4:in `require'
...
```

On ruby 2.0.0:
```
/home/melissa/puppetlabs-cloud_provisioner/lib/puppet/cloudpack.rb:5:in `require': cannot load such file -- net/ssh (LoadError)
	from /home/melissa/puppetlabs-cloud_provisioner/lib/puppet/cloudpack.rb:5:in `<top (required)>'
	from /home/melissa/puppetlabs-cloud_provisioner/spec/spec_helper.rb:15:in `require'
	from /home/melissa/puppetlabs-cloud_provisioner/spec/spec_helper.rb:15:in `<top (required)>'
...
```

Do you want me to try to address those failures as well?